src/clr-angular/utils/_mixins.clarity.scss

```diff
@@ -95,6 +95,27 @@ $clr-outline-spread: 2px;
   }
 }
 
+@mixin equilateral($size: 1.2rem) {
+  height: $size;
+  width: $size;
+}
+
+@mixin forcemin-equilateral($size: 1.2rem) {
+  @include equilateral($size);
+  min-height: $size;
+  min-width: $size;
+}
+
+@mixin forcemax-equilateral($size: 1.2rem) {
+  @include equilateral($size);
+  max-height: $size;
+  max-width: $size;
+}
+
+@mixin size-zero() {
```

Comment:
May be a nit, but seems unnecessary to have a specialized method for zero.

Reply:
Makes sense. I'll get rid of it.